xbmc/video/VideoDatabase.h

```diff
@@ -513,11 +513,12 @@ class CVideoDatabase : public CDatabase
   bool LoadVideoInfo(const std::string& strFilenameAndPath, CVideoInfoTag& details, int getDetails = VideoDbDetailsAll);
   bool GetMovieInfo(const std::string& strFilenameAndPath, CVideoInfoTag& details, int idMovie = -1, int getDetails = VideoDbDetailsAll);
   bool GetTvShowInfo(const std::string& strPath, CVideoInfoTag& details, int idTvShow = -1, CFileItem* item = NULL, int getDetails = VideoDbDetailsAll);
+  bool GetSeasonInfo(int idSeason, CVideoInfoTag& details, CFileItem* item);
```

Comment:
Isn't it a better option to overload the  `GetSeasonInfo` and `GetSetInfo` providing new methods:

`GetSeasonInfo(int idSeason, CFileItem* item)`
`GetSetInfo(int idSet, CFileItem* item)`

in line with the already existing ones? IMHO would make the interface/contract a bit more clean than those "dummy" item path you provide.

Reply:
Re `bool GetSetInfo(int idSet, CVideoInfoTag& details, CFileItem* item = nullptr);`: This is a new method, following the signature pattern for the other Get-ters, even the option itemptr param. No "dummy" involved here, not even in the implementation.